File: 251002/Voevoda/internal/entity/tweet.go
package entity

import "time"

type Tweet struct {
	ID       int64     `json:"id,omitempty"`
	UserID   int       `json:"userId"`
	Title    string    `json:"title"`
	Content  string    `json:"content"`
	Created  time.Time `json:"created"`
	Modified time.Time `json:"modified"`
}

File: 251003/Novichenko/RV1/src/main/java/com/example/rv1/entity/Tweet.java
package com.example.rv1.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Tweet {
    private int id;
    private int userId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;
}
